geomesa-process/src/main/scala/org/locationtech/geomesa/process/HashAttributeProcess.scala

```diff
@@ -42,7 +43,7 @@ trait HashAttribute {
         featureBuilder.reset()
         featureBuilder.init(sf)
         val attr = Option(sf.getAttribute(attribute)).map(_.toString).getOrElse("")
-        val hash = math.abs(hashFn.hashString(attr).asInt()) % modulo
+        val hash = math.abs(hashFn.hashString(attr, Charsets.UTF_16LE).asInt()) % modulo
```

Comment:
It was previously UTF_16LE, they just removed a method that defaulted it to it, so it must be stated explicitly now (http://docs.guava-libraries.googlecode.com/git-history/v14.0.1/javadoc/com/google/common/hash/HashFunction.html)